modules.d/95virtfs/module-setup.sh

```diff
@@ -2,7 +2,7 @@
 
 # called by dracut
 check() {
-    [[ $hostonly ]] || [[ $mount_needs ]] && {
+    [[ $hostonly_mode == "strict" ]] || [[ $mount_needs ]] && {
         for fs in "${host_fs_types[@]}"; do
             [[ $fs == "9p" ]] && return 0
         done
```

Comment:
The whole check function in this file seems a bit weird - it defaults to not including 9pfs support, but in hostonly mode it might include it if the system uses 9pfs, but now it will only follow that in strict hostonly mode. I'm not really sure I understand the intent here, and I suspect this will regress use cases that  depend on 9pfs in the wild.

Maybe this should be changed to match the 95virtiofs module's logic, which includes the module by default and has the `is_qemu_virtualized` check in the hostonly_strict block.